client/src/components/CompetitionNavigation/CompetitionNavigation.jsx

```diff
@@ -59,6 +60,7 @@ function CompetitionNavigation() {
         <Route path="rounds/:roundId/*" element={<Round />} />
         <Route path="competitors" element={<Competitors />} />
         <Route path="competitors/:competitorId" element={<Competitor />} />
+        <Route path="records" element={<CompetitionRecords />} />
```

Comment:
I think we should have put it directly on the competition page under schedule, but display the section only if there is any record. Otherwise people may click on the icon just to find out there are no records, and also having it on the main page makes the record stand out, which makes sense.